### PR TITLE
tokens are like a box of chocolates

### PR DESCRIPTION
**Pull request by melmsie**

### CODE REVIEW

File: src/Powercord/index.js [hunk 1]
```diff
@@ -49,6 +50,10 @@ module.exports = class Powercord extends EventEmitter {
         SettingsManager.upload();
       }
     });
+
+    if (this.settings.get('hideToken')) {
```
Comment (cyyynthia) [inline src/Powercord/index.js]: Don't forget default value
```suggestion
    if (this.settings.get('hideToken', true)) {
```

File: src/Powercord/index.js [hunk 2]
```diff
@@ -36,6 +36,7 @@ module.exports = class Powercord extends EventEmitter {
   }
 
   async init () {
+
```
Comment (cyyynthia) [inline src/Powercord/index.js]: ok

**Review by cyyynthia — CHANGES_REQUESTED** [review]

File: src/Powercord/index.js (hunk 2 above)
Comment (melmsie) [reply src/Powercord/index.js]: Listen I left this out on purpose on the first commit, and forgot to revert it before the second commit

File: src/Powercord/index.js [hunk 3]
```diff
@@ -49,6 +50,10 @@ module.exports = class Powercord extends EventEmitter {
         SettingsManager.upload();
       }
     });
+
+    if (this.settings.get('hideToken')) {
+      require('powercord/webpack').getModule(['hideToken']).hideToken = () => void 0
```
Comment (21Joakim) [inline src/Powercord/index.js]: Shouldn't it be [ 'hideToken' ] because lintttttttttttttttttttt

File: src/Powercord/index.js (hunk 2 above)
Comment (melmsie) [reply src/Powercord/index.js]: It's been a long ass day and my mother in law is coming over soon, so my head is in a different place

File: src/Powercord/index.js (hunk 3 above)
Comment (davidcralph) [reply src/Powercord/index.js]: lol lint

Comment (melmsie) [reply src/Powercord/index.js]: lol lint

Comment (cyyynthia) [reply src/Powercord/index.js]: lol lint

Comment (MrRobotjs) [reply src/Powercord/index.js]: lol lint

Comment (MrLar) [reply src/Powercord/index.js]: lol lint

Comment (melmsie) [reply src/Powercord/index.js]: lol linted

**Review by cyyynthia — APPROVED** [review]
ok linted

**Pull request merged**